static/50x.html

```diff
@@ -9,6 +9,9 @@
     <link type="text/css" rel="stylesheet" href="/static/css/main.css">
     <link type="text/css" rel="stylesheet" href="/static/css/account.css">
     <link rel="apple-touch-icon" href="/static/images/apple-touch-icon.png">
+    <!-- Behave more like an app when saved to home screen in iOS -->
```

Comment:
It *may* only be needed for `base.html`, but I added it here for completeness. I suppose there's a possibility that you've saved the app to the home screen and you get a 500 next time you open it. In that case, it seems like these additions would apply.